packages/cli/package.json

```diff
@@ -85,5 +92,11 @@
   },
   "bin": {
     "zapier": "zapier.js"
+  },
+  "oclif": {
+    "commands": "./src/ocommands",
+    "plugins": [
+      "@oclif/plugin-not-found"
```

Comment:
this provides our "`buil` is not a command. did you mean `build`?" out of the box!